src/Microsoft.Azure.WebJobs.Host/Executors/JobHostContext.cs

```diff
@@ -17,20 +18,26 @@ internal sealed class JobHostContext : IDisposable
         private readonly IListener _listener;
         private readonly TraceWriter _trace;
         private readonly IAsyncCollector<FunctionInstanceLogEntry> _fastLogger; // optional
+        private readonly IAsyncCollector<FunctionResultLog> _aggregator;
```

Comment:
Do we need both of these? can we get rid of _fastLogger and just use _aggregator?